chart/templates/ws-manager-bridge-deployment.yaml

```
@@ -35,6 +35,7 @@ spec:
         kind: pod
         stage: {{ .Values.installation.stage }}
     spec:
+      priorityClassName: system-node-critical
```

```suggestion
      priorityClassName: system-cluster-critical
```

(requires an additional resource quota)